centreon/packages/ui/src/components/DataTable/Item/DataTableItem.tsx

```diff
@@ -27,7 +28,8 @@ const DataTableItem = forwardRef(
       hasCardAction = false,
       hasActions = false,
       onClick,
-      Actions
+      Actions,
+      fovoriteAction
```

Comment:
```suggestion
```
Why did you create a prop for favorite actions as the Favorite icon is already in the Actions prop